diff --git a/usr/usr/bin/hw-management.sh b/usr/usr/bin/hw-management.sh
index 66248504..2bfd2943 100755
--- a/usr/usr/bin/hw-management.sh
+++ b/usr/usr/bin/hw-management.sh
@@ -2718,8 +2718,6 @@ create_symbolic_links()
 	fi
 	if [ ! -d $bin_path ]; then
 		mkdir $bin_path
-		# Copy binaries to make them available for the access from containers.
-		cp /usr/bin/hw_management_independent_mode_update.py "$bin_path"
 	fi
 	if [ ! -d $dynamic_boards_path ]; then
 		mkdir $dynamic_boards_path
@@ -3093,8 +3091,6 @@ do_start()
 	touch $udev_ready
 	depmod -a 2>/dev/null
 	set_config_data
-	udevadm trigger --action=add
-	udevadm settle
 	set_sodimm_temp_limits
 	set_gpios "export"
 	create_event_files
@@ -3127,11 +3123,6 @@ do_start()
 	if [ -v "lm_sensors_labels" ] && [ -f $lm_sensors_labels ]; then 
 		ln -sf $lm_sensors_labels $config_path/lm_sensors_labels
 	fi 
-	if [ -v "thermal_control_config" ] && [ -f $thermal_control_config ]; then
-		cp $thermal_control_config $config_path/tc_config.json
-	else
-		cp $thermal_control_configs_path/tc_config_default.json $config_path/tc_config.json
-	fi
 	log_info "Init completed."
 }
 
