diff --git a/configure.ac b/configure.ac
index 25b8d3a..faac14f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -90,13 +90,10 @@ fi
 AM_CONDITIONAL(KERNEL_BUILD,[test ${KERNEL_BUILD_CHK} = 1])
 AC_SUBST(KERNEL_BUILD)
 
-LDFLAGS="${LDFLAGS} -L/usr/local/lib -L/usr/local/lib64 -L/usr/lib64"
-CXXFLAGS="${CXXFLAGS} -isystem /usr/local/include" 
 if test "x$OS" = "xFreeBSD"; then
     AC_MSG_NOTICE(FreeBSD Build)
     MTCR_CONF_DIR="mtcr_freebsd"
     default_en_inband="no"
-    CFLAGS="${CFLAGS} -I /usr/local/include"
 else
     MTCR_CONF_DIR="mtcr_ul"
     LDL="-ldl"
